src/routes/PageRoutes.js

```diff
@@ -53,8 +53,15 @@ export const PageRoutes = () => {
           </RequiresAuth>
         }
       />
+      <Route
+        path="/profile"
+        element={
+          <RequiresAuth>
+            <MyProfile />
+          </RequiresAuth>
```

Comment:
You can try to implemented nested conditional routes

Reply:
Thanks for Review, will implement it later.